requirements.txt

```diff
@@ -0,0 +1,5 @@
+fonttools==3.0
```

Comment:
I know Dave, but it's 3 months old and quite behind the current HEAD. OK,
we shall discuss on the upstream repo whether we need to tag a new 3.1
release.
Il 13/nov/2015 08:13, "Dave Crossland" <notifications@github.com> ha
scritto:

> In requirements.txt
> <https://github.com/trufont/trufont/pull/140#discussion_r44758409>:
>
> > @@ -0,0 +1,5 @@
> > +fonttools==3.0
>
> Its the latest release, https://github.com/behdad/fonttools/releases
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trufont/trufont/pull/140/files#r44758409>.
>